.changelog/1387.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+internal: Add support for Deployment URLs
```

Comment:
We don't add internal changes to the changelog, recommendation below:

```suggestion
core: platform plugins may now advertise deployment-specific URLs
```

Reply:
I have no clue on why it is not working for me (using Nix as well). A double hypen fixes the issue for me, but as we already discussed this should be addressed in another PR. Sorry, it might have slipped into my commit.